.github/workflows/publish-helm-chart.yaml

```diff
@@ -21,6 +23,20 @@ jobs:
         uses: azure/setup-helm@v1
         with:
           version: v3.5.0
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Setup Chart Linting
+        id: lint
+        uses: helm/chart-testing-action@v2.1.0
+      - name: Setup Kubernetes cluster (KIND)
+        uses: engineerd/setup-kind@v0.5.0
```

Comment:
Why don't we use https://github.com/helm/kind-action instead of engineerd/setup-kind?